raco/representation.py

```diff
@@ -0,0 +1,17 @@
+__author__ = 'brandon'
+
+
+class RepresentationProperties(object):
+    def __init__(self, hash_partitioned=set(), sorted=None, grouped=None):
```

Comment:
Why is `hash_partitioned` an empty collection by default but the others are not?